Libs/Scripting/Python/Core/ctkAbstractPythonManager.cpp

```diff
@@ -520,8 +534,16 @@ QStringList ctkAbstractPythonManager::pythonAttributes(const QString& pythonVari
           line_code.append('.'); // add the point again in case we need to continue to fill line_code
           object = PyObject_GetAttrString(main_object,instantiated_class_name.toLatin1().data());
 
-          dict = object;
-          results = ctkAbstractPythonManager::dir_object(object,appendParenthesis);
+          if (object)
+          {
+            results = ctkAbstractPythonManager::dir_object(object, appendParenthesis, excludeGetSet);
+          }
+          else
+          {
+            // failed to invoke callable, no completions are available
+            PyErr_Clear();
```

Comment:
```suggestion
            PythonQt::self()->handleErrorAsWarning();
```

Instead of blindly discarding error ... we could introduce a new function `handleErrorAsWarning` allowing those to be logged while avoiding them to be reported in the Python console.

Reply:
Attempt to replace the calls `PyErr_Clear();` with `PythonQt::self()->handleError();` allows to identify the "problematic" properties but polluted the output:

![image](https://github.com/user-attachments/assets/c6f0e6b5-9c3e-4da2-86c5-ff68c5cfc3e8)